.github/workflows/build.yml

```diff
@@ -78,6 +78,69 @@ jobs:
           if-no-files-found: error
           retention-days: 7
 
+  ubuntu18:
+    strategy:
+      matrix:
+        arch:
+          [
+          "linux/amd64 x86_64"
+          ]
+    name: "Packaging for Ubuntu 18.04"
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
+        with:
+          platforms: all
+      - name: Set up Docker Buildx
+        id: buildx
+        uses: docker/setup-buildx-action@v2
+        with:
+          version: latest
+      - name: Read info
+        id: tags
+        shell: bash
+        run: |
+          arch="${{ matrix.arch }}"
+          echo "PLATFORM=${arch%% *}" >> "$GITHUB_OUTPUT"
+          echo "ARCH=${arch##* }" >> "$GITHUB_OUTPUT"
+      - name: Installing xmllint for ci-set-vars
+        run: sudo apt -qy install libxml2-utils
+      - name: set environment variables
+        id: set_vars
+        run: ./scripts/ci-set-vars.sh
+        env:
+          REPOSITORY: ${{ github.event.repository.name }}
+      - name: Build ${{ matrix.arch }} release
+        shell: bash
+        run: |
+          docker buildx build --platform ${{ steps.tags.outputs.PLATFORM }} \
+          --tag contour:${{ steps.tags.outputs.ARCH }} \
+          -f .github/ubuntu18/Dockerfile \
+          --load \
+          .
+          docker create --name contour-${{ steps.tags.outputs.ARCH }} contour:${{ steps.tags.outputs.ARCH }}
+          docker cp contour-${{ steps.tags.outputs.ARCH }}:/home/docker/contour/package.deb \
+          "contour-${{ steps.set_vars.outputs.VERSION_STRING }}-ubuntu18.04-amd64.deb"
+          docker cp contour-${{ steps.tags.outputs.ARCH }}:/home/docker/contour/package.ddeb \
+          "contour-dbgsym-${{ steps.set_vars.outputs.VERSION_STRING }}-ubuntu18.04-amd64.ddeb"
```

Comment:
Veeery nice. I was afraid that this would be more complicated to get them out of there. :)